### PR TITLE
Fixture for running couchbase workload

### PR DESCRIPTION
**Pull request by shreekarSS**
- Created fixture for running couchbase workload

Signed-off-by: Shreekar <sshreeka@redhat.com>

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -2470,6 +2471,40 @@ def finalizer():
...
+    def factory(replicas=3):
+        """
+        Factory to start couchbase workload
+
+        Args:
+            replicas (int): Number of couchbase workers to be deployed
+        """
+        # Setup couchbase
+        couchbase.setup_cb()
+        # Create couchbur Dase workers
```
Comment (PrasadDesala) [inline tests/conftest.py:2490]: Typo here, please correct. It should be couchbase

**Comment by PrasadDesala** [comment]
You have added label Verified to the PR, but I don't see any jenkin pr validation run. 
Please check

**Comment by shreekarSS** [comment]
> You have added label Verified to the PR, but I don't see any jenkin pr validation run.
> Please check

verified locally, did not create any test case for that

File: tests/conftest.py (hunk 1 above)
Comment (shreekarSS) [reply tests/conftest.py]: done

**Pull request merged**